spot-client/src/spot-tv/ui/components/setup/SyncWithBackend.js

```diff
@@ -34,12 +38,22 @@ export class SyncWithBackend extends React.Component {
         super(props);
 
         this.state = {
+            autoSyncing: false,
```

Comment:
Shouldn't this default to true? It seems like it defaults to false and just gets set to true immediately after mount. What if its default is based on if there is an "llpc"?

Reply:
Yeah, I was thinking about it defaulting to true. Let me revisit this and see if I can improve.

Reply:
Done, thanks!